qiskit/providers/aer/extensions/snapshot_expectation_value.py

```diff
@@ -18,6 +18,7 @@
 from qiskit import QuantumCircuit
 from qiskit.circuit import Instruction
 from qiskit.extensions.exceptions import ExtensionError
+from qiskit.qobj.qasm_qobj import QasmQobjInstruction
```

Comment:
Want to wait for ci to finish, but this should be changed to:

```suggestion
from qiskit.qobj import QasmQobjInstruction
```
so that this will work for 0.12.0 and 0.13.0